tool/fragment/pom.xml

```diff
@@ -63,7 +63,6 @@
             <groupId>${project.groupId}</groupId>
             <artifactId>cdk-data</artifactId>
             <version>${project.parent.version}</version>
-            <scope>test</scope>
```

Comment:
cdk-data should not be in the non-test scope. Generally you may need a builder passed in you get it from the provided molecule/container.

Reply:
I used "instanceof PseudoAtom" to detect pseudo atom instances. This was my only non-test use of cdk-data. I've adjusted this in my last commit (https://github.com/cdk/cdk/pull/1039/commits/a19085ef204925cbf8135a0498094b16cb3fecd0) and re-assigned the test scope to cdk-data.